src/cpp/session/modules/SessionRParser.cpp

```diff
@@ -2225,24 +2225,21 @@ void doParse(RTokenCursor& cursor,
       
       if (cursor.isType(RToken::COMMA))
       {
-         if (status.parseOptions().checkArgumentsToRFunctionCalls())
```

Comment:
This is now checked only prior to the `validateFunctionCall()` call here: https://github.com/rstudio/rstudio/blob/master/src/cpp/session/modules/SessionRParser.cpp#L2177-L2178

For these checks, since they're purely syntactic I thought it made sense to have them on as part of the 'default' completions.